app/controllers/api/verify/document_capture_errors_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
+  module Verify
+    class DocumentCaptureErrorsController < BaseController
+      include EffectiveUser
+
+      self.required_step = nil
+
+      def delete
+        form = DocumentCaptureErrorsDeleteForm.new(
+          document_capture_session_uuid: params[:document_capture_session_uuid],
+        )
+        result, document_capture_session = form.submit
+
+        if result.success?
+          document_capture_session.update(ocr_confirmation_pending: false)
+          render json: {}
```

Comment:
a completely empty JSON object for success to me seems hard to interpret. I feel like we could do a 204 (empty) with an empty body, or just throw in a small boolean?
```suggestion
          render json: { success: true }
```

Reply:
Yeah, that's fair. I was leaning a bit heavy on status codes for indicating success here. I don't feel 100% about the full consequences of a 204 status with an empty body, so may just shove in the boolean for pragmatism's sake.

Reply:
Another reason for shying away from a `success` value was to avoid ambiguity for how we're considering an "error" response in other cases.

For example:

https://github.com/18F/identity-idp/blob/8f6833934b1a94b305b2d4266a51cc01a1d2314a/app/javascript/packages/verify-flow/services/api.ts#L49-L51

Reply:
Yeah good point, I guess we don't have super strict API definitions/standards so I don't mean to get too off track here, happy to leave as-is too